css.js

```diff
@@ -143,9 +145,11 @@ class CSSTask extends TaskKitTask {
         mixins: this.mixins,
         mixinsFiles: this.options.mixinPath
       }),
+      csseasings(),
       inlinesvg(),
       svgo(),
       triangle(),
+      cssnested(),
       cssnext({
```

Comment:
probably need to turn off the css next nesting

Reply:
Good flag, wasn't aware